pillar.example

```diff
@@ -42,6 +42,33 @@ zabbix-agent:
   extra_conf: |
     # Here we can set extra agent configuration lines
 
+## Zabbix Agent for Windows ##
+zabbix-agent:
+  server:
+    - zabbix.example.com
+  serveractive:
+    - localhost
+  listenip: 0.0.0.0
+  listenport: 10050
+  hostmetadata: c9767034-22c6-4d3d-a886-5fcaf1386b77
+  # For zabbix-agent below version 3 if you want to use syslog instead of file specify
+  # logfile: syslog
+  logfile: 'C:\program files\zabbix agent\zabbix_agentd.log'
+  logfilesize: 0
```

Comment:
Afaik there is no `logrotate` on Windows and it will be useful to let agent rotate log files by itself.
```suggestion
  logfilesize: 5
```